src/components/UserSelector.tsx

```diff
@@ -19,23 +43,20 @@
       </div>
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
-        <div className="dropdown-content">
-          <a href="#user-1" className="dropdown-item">
-            Leanne Graham
-          </a>
-          <a href="#user-2" className="dropdown-item is-active">
-            Ervin Howell
-          </a>
-          <a href="#user-3" className="dropdown-item">
-            Clementine Bauch
-          </a>
-          <a href="#user-4" className="dropdown-item">
-            Patricia Lebsack
-          </a>
-          <a href="#user-5" className="dropdown-item">
-            Chelsey Dietrich
-          </a>
-        </div>
+        {showUserList && (
+          <div className="dropdown-content">
+            {users.map(user => (
+              <a
```

Comment:
Using `href` with a hash (`#`) can cause the page to jump to the top when clicked. Consider using a `button` element or `role="button"` on the anchor tag to prevent this behavior.